src/devices/gmmts.ts

```diff
@@ -0,0 +1,848 @@
+import {Definition, Fz, Tz, KeyValue, Zh} from '../lib/types';
+/* eslint-disable camelcase */
+/* eslint-disable max-len */
```

Comment:
Please don't do any `eslint-disable` here

Reply:
Where I can use it ?

Reply:
Just remove it

Reply:
Hi,
If I remove them, I have a lot of lint errors:
![image](https://github.com/Koenkk/zigbee-herdsman-converters/assets/47485034/a36a125f-0052-46a7-b721-e0f9f677fbc4)

If I remove spaces; and cut the line at 150 characters; the code becomes very ugly.
Can I keep at least 
/* eslint-disable max-len */
/* eslint-disable no-multi-spaces */ 
?
Thanks

Reply:
Then let's keep those